gc/gc-base/pom.xml

```diff
@@ -41,6 +41,12 @@
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
+    <!--- We cannot use version > 0.43 beacuse it uses a kryo version 5.x that is incompatible with out spark (requires 4.x) -->
+    <dependency>
+      <groupId>de.javakaffee</groupId>
+      <artifactId>kryo-serializers</artifactId>
+      <version>0.43</version>
```

Comment:
can we move this out into a property within the root `pom.xml` and also define the dependency in the `dependencyManagement` section?

Reply:
do you want me to move all dependencies from this module to the new `dependencyManagement` section or only this one?